src/Domain/DoZendesk.php

```diff
@@ -2,18 +2,30 @@
 namespace Wheniwork\Feedback\Domain;
 
 use RuntimeException;
+use Wheniwork\Feedback\Service\Authorizer;
+use Wheniwork\Feedback\Service\GithubService;
+use Wheniwork\Feedback\Service\HipChatService;
 
 class DoZendesk extends FeedbackDomain
 {
+    private $auth;
+
+    public function __construct(
+        HipChatService $hipchat,
+        GithubService $github,
+        Authorizer $auth
+    ) {
+        parent::__construct($hipchat, $github);
+        $this->auth = $auth;
```

Comment:
Looks like you're repeating the auth assignment, so maybe put the auth in the `FeedbackDomain`? If you did so, you could keep the `private $auth` by providing a method in the parent to handle it.